content/en/apps/guides/performance/telemetry.md

```diff
@@ -90,6 +90,8 @@ The telemetry data gathered changes with different versions of the framework. Cu
 | `enketo:<training-card>:add:user_edit_time` | The time the user took to complete the training card. Added in 4.2.0 |
 | `enketo:<training-card>:add:save` | The time it took to save the training card. Added in 4.2.0 |
 | `enketo:<training-card>:add:quit` | The time from when the training card was rendered to when the user quits the training. Added in 4.2.0 |
+| `geolocation:success` | A successful GPS response with the value showing the accuracy. |
+| `geolocation:failure:-1` | An unsuccessful GPS response. This may indicate GPS is disabled on the device, or the service took too long determine the position of the device. |
```

Comment:
```suggestion
| `geolocation:failure:<x>` | An unsuccessful GPS response. `x` is a constant matching the [GeolocationPositionError](https://developer.mozilla.org/en-US/docs/Web/API/GeolocationPositionError/code) or with one of the following values: `-1` unknown failure, `-2` timeout, or `-3` geolocation services unavailable. |
```

Reply:
dunno if this is too dense? i can make a new PR for -2/-3 but I do think it's important to explain the positive error codes

Reply:
It's quite dense but really useful, thanks!